bears/installation_tool.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import importlib
+import re
+import subprocess
+import sys
+
+from coala_utils.Question import ask_question
+from coalib.collecting.Collectors import get_all_bears_names
+from coalib.collecting.Importers import iimport_objects
+from coalib.misc.Shell import call_without_output
+from prompt_toolkit import prompt
+from prompt_toolkit.contrib.completers import WordCompleter
+from pygments.lexers import SqlLexer
+
+
+def install_pip_package(package_name):
+    """
+    Uses call to install a PyPi package.
+
+    :param package_name: The package to be installed.
+    """
+    print('To install the dependencies correctly, we will need administrator '
+          'rights from you.')
+    subprocess.call(['sudo', sys.executable, '-m',
+                     'pip', 'install', package_name, '--upgrade'])
+
+
+def get_output(command):
+    """
+    Runs the command and decodes the output and returns it.
+
+    :param command: The command to be run.
+    :return:        The output of the command, decoded.
+    """
+    function = subprocess.Popen(command, stdout=subprocess.PIPE)
+    output, err = function.communicate()
+    return output.decode("utf-8")
```

Comment:
He's not using it anyway. If he's still going for the unpack, may be do `output, _ = ...`? Or at least `_err`?